scripts/utils.ts

```diff
@@ -360,7 +361,7 @@ export async function checkInstanceHealth(instanceId: string, region: string = "
       }
 
       let attempts = 0;
-      const maxAttempts = 40;
+      const maxAttempts = 20;
```

Comment:
Make this update in `waitForInstanceHealthy` in `create-agent` too, reducint the waitTime _there_ to always be 1 second instead of 5-10